src/menu/track.cpp

```diff
@@ -451,8 +451,8 @@ auto Menu::Track::getTrackUrl() const -> QString
 		return {};
 	}
 
-	const auto trackUri = lib::spt::api::to_uri("tracks", tracks.cbegin()->second.id);
-	auto str = lib::fmt::format("https://open.spotify.com/track/{}", trackUri);
+	const auto trackId = lib::spt::api::to_id(tracks.cbegin()->second.id);
```

Comment:
`lib::spt::api::to_id` shouldn't be needed, as `lib::spt::track::id` should always be an ID. I'm also not really sure if sharing should be available when selecting multiple tracks, as it's not really obvious what you're sharing, but that's an issue for later.

```suggestion
	const auto trackId = tracks.cbegin()->second.id;
```